### PR TITLE
Swift 5.4 docker setup

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -17,9 +17,11 @@ RUN apt-get update && apt-get install -y wget
 RUN apt-get update && apt-get install -y lsof dnsutils netcat-openbsd net-tools curl jq # used by integration tests
 
 # ruby and jazzy for docs generation
-RUN apt-get update && apt-get install -y ruby ruby-dev libsqlite3-dev
+RUN apt-get update && apt-get install -y ruby ruby-dev libsqlite3-dev build-essential
+# switch of gem docs building
+RUN echo "gem: --no-document" > ~/.gemrc
 # jazzy no longer works on xenial as ruby is too old.
-RUN if [ "${ubuntu_version}" != "xenial" ] ; then  gem install jazzy --no-ri --no-rdoc ; fi
```
Comment (weissi) [inline docker/Dockerfile:22]: `--no-ri --no-rdoc` don't work anymore. Stackoverflow suggests the [gemrc thing](https://stackoverflow.com/questions/1381725/how-to-make-no-ri-no-rdoc-the-default-for-gem-install).

Comment (tomerd) [reply docker/Dockerfile:22]: https://github.com/jdowning/puppet-rbenv/issues/80

Comment (weissi) [reply docker/Dockerfile:22]: @tomerd yes, that's what I did (using `gemrc`)

**Review by Lukasa — APPROVED** [review]

**Comment by tomerd** [comment]
@weissi added 5.4 job, which is failing at this time. I am fine with merging this tho.

**Pull request merged**